cli/helptext.go

```diff
@@ -225,9 +246,9 @@ func ShortHelp(rootName string, root *cmds.Command, path []string, out io.Writer
 func generateSynopsis(cmd *cmds.Command, path string) string {
 	res := path
 	for _, opt := range cmd.Options {
-		valopt, ok := cmd.Helptext.SynopsisOptionsValues[opt.Names()[0]]
+		valopt, ok := cmd.Helptext.SynopsisOptionsValues[opt.Name()]
 		if !ok {
-			valopt = opt.Names()[0]
+			valopt = opt.Name()
```

Comment:
We only have one name for options now?

Reply:
We have one main name and `n` aliases. That way you always use the long form when accessing the option in your code, but you can set the option using any of the provided names.

Reply:
That made the cmds code simpler and the actual commands easier to read because you have a consistent name for every option.